### PR TITLE
Add code lint configuration

### PR DESCRIPTION
**Pull request by marcospereira**
Add ESLint to the project so that we can have some code validation
when submitting pull requests.

### CODE REVIEW

File: aws/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "akka-cloud-platform-deploy",
     "devDependencies": {
-        "@types/node": "^15.0.2"
+        "@types/node": "^14.0.0"
```
Comment (marcospereira) [inline aws/package.json:4]: Node 14 is the current LTS version.

File: gcp/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "akka-cloud-platform-gcp-deploy",
     "devDependencies": {
-        "@types/node": "^10.0.0"
+        "@types/node": "^14.0.0"
```
Comment (marcospereira) [inline gcp/package.json:4]: sync with the version used by `aws`.

**Review by RayRoestenburg — APPROVED** [review]
LGTM

**Review by marcospereira — COMMENTED** [review]
- Declare a bunch of variables as `const` instead of `let`
- Remove unused variables

**Pull request merged**